sugaroid/brain/preprocessors.py

```diff
@@ -62,6 +62,7 @@
     "btw":"by the way",
     "becoz":"because",
     "yaya":"yea",
+    "yea":"yes",
```

Comment:
Do you know these changes are conflicting?

`yaya` is changed to `yea`, but `yea` is changed to `yes`

So eventually, `yaya` is changed to `yea`, so no net benefit.

Reply:
@srevinsaju, I thought that `yea` is not defined as it returned a random response from `oneword`'s constants while `yes` returns response from yesno.py

```
( ဖ‿ဖ) @> yea
sugaroid: Sometimes, one or two words does not help me understand
```

```
( ဖ‿ဖ) @> yes
sugaroid: I shall annoy you. A big NO
```

These are the responses I receive for `yea` and `yes` respectively!  If `yaya` is not eventually changed to `yes` in this case,  it would return an annoying response requesting a 'proper sentence structure'.

Reply:
what?